pycln/utils/refactor.py

```diff
@@ -20,6 +20,13 @@
 from .config import Config
 from .report import Report
 
+if sys.version_info >= (3, 12):
+    from pathlib import Path
+    _flavour = os.path
+else:
+    from pathlib import Path, _posix_flavour, _windows_flavour  # type: ignore
+    _flavour = _windows_flavour if ISWIN else _posix_flavour
```

Comment:
I think the `if` should be reversed. Versions *prior* to 3.12 are an exception, not the other way around.